cvmfs/statistics_database.h

```diff
@@ -11,6 +11,7 @@
 #include "options.h"
 #include "sql.h"
 #include "statistics.h"
+#include "swissknife.h"
```

Comment:
If you are not storing dry runs in the database, there is no need to include `swissknife.h` here. The `args` parameter of the `StoreStatistics` function can also go.

A design point, for future reference: if we weren't removing the `args` parameter, we would still improve the function by observing that a single argument from the `args` list is used (dry_run in this case). It's better to read the value of the argument in the caller function and have the `StoreStatistics` function receive the value directly. This way, we don't need to be aware of any type from  `swissknife.h` in this file and we don't need to include it.
You really want to avoid complicated include relationships between your files: swissknife uses the `StatisticsDatabase` class, this class shouldn't need to know anything about it's users.